app/assets/javascripts/winners-bundle.js

```diff
@@ -1,4 +1,5 @@
-//= require d3
-//= require winners/c3
-//= require winners/underscore
-//= require_tree ./winners/
```

Comment:
If both work, I would vote `require_tree` explicitly listing, them, but I don't have a strong opinion either way. Completely fine with this.